meson.build

```diff
@@ -1714,6 +1729,20 @@ install_libudev_static = static_library(
         c_args : static_libudev_pic ? [] : ['-fno-PIC'],
         pic : static_libudev_pic)
 
+if conf.get('HAVE_LIBCRYPTSETUP_PLUGINS') == 1
+        cryptsetup_token_systemd_tpm2 = shared_library(
```

Comment:
Any specific reason to make this a new internal shared object, rather than a static one linked into libsystemd-shared?

Reply:
libcryptsetup looks specifically for "libcryptsetup-token-<token_name>.so". It is used by both libcryptsetup and systemd utilities. It's a whole purpose of this PR.

So that we can unlock and manipulate (resize, reload, etc) devices with systemd-tpm2 token from both cryptsetup and systemd cmd line utilities.

Reply:
Ok, I had missed that it needs to be a shared object that libcryptsetup loads, thanks